internal/main.go

```diff
@@ -0,0 +1,240 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+	"sync"
+
+	"github.com/gin-gonic/gin"
+	"github.com/lib/pq"
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+	"gorm.io/gorm/clause"
+)
+
+type DBMedia struct {
```

Comment:
Could these types/structs be extracted to separate files?

Reply:
Done